.github/workflows/build-test.yml

```diff
@@ -466,17 +466,7 @@ jobs:
       # debugpy is not shipped, only installed for local tests.
       # In production, we get debugpy from python extension.
       - name: Install Python Libs
-        if: matrix.python != 'conda' && matrix.python != 'noPython' && matrix.tags == '^[^@]+$|@mandatory'
-        run: |
-          python --version
-          python -c "import sys;print(sys.executable)"
-          python -m pip --disable-pip-version-check install -t ./pythonFiles/lib/python --no-cache-dir --implementation py --no-deps --upgrade -r ./requirements.txt
-          python -m pip --disable-pip-version-check install ipykernel
-          python ./pythonFiles/install_debugpy.py
-          python -m ipykernel install --user
-
-      - name: Install Python Libs
```

Comment:
The remote tests were failing, i'm restoring the previous CI job, will address this in debt week